src/envo/envo-edit.owl

```diff
@@ -44974,6 +44975,17 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03605005> "epixylon"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03605005> <http://purl.obolibrary.org/obo/ENVO_03605000>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_03605006> (stream water)
+
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03605006> "ORCID:0000-0002-4366-3088")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#creation_date> <http://purl.obolibrary.org/obo/ENVO_03605006> "2024-01-12T13:48:00Z")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasOBONamespace> <http://purl.obolibrary.org/obo/ENVO_03605006> "ENVO")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://purl.obolibrary.org/obo/ENVO_03605006> "ENVO:03605006")
```

Comment:
no need to do this busy work (but fine to leave in)

Reply:
yes, we generally don't add the namespace and CURIE style IDs anymore. Even removing them as they're redundant with the IRI information